scripts/promptflow-ci/promptflow_ci.py

```diff
@@ -165,7 +165,14 @@ def check_flow_run_status(
     parser.add_argument('--flow_submit_mode', type=str, default="sync")
     parser.add_argument('--run_time', type=str, default="default-mir")
     parser.add_argument('--skipped_flows', type=str,
-                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow")
+                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow,playground-ayod-rag,"
+                        "chat-quality-safety-eval,qna-non-rag-metrics-eval,qna-quality-safety-eval,"
+                        "qna-rag-metrics-eval,rai-eval-ui-dag-flow,rai-qna-quality-safety-eval,"
+                        "qna-ada-similarity-eval,qna-relevance-eval,ask-wikipedia,classification-accuracy-evals,"
```

Comment:
classification-accuracy-eval**s** change to classification-accuracy-eval, without s